airrohr-firmware/airrohr-firmware.ino

```diff
@@ -1,5 +1,5 @@
 #include <Arduino.h>
-#define INTL_DE
+#define INTL_NL
```

Comment:
Please do not change the language on your commit :)

Reply:
OK